model/registry/test.yaml

```diff
@@ -0,0 +1,72 @@
+groups:
+  - id: registry.test.suite
+    prefix: test.suite
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to
+     [software tests](https://en.wikipedia.org/wiki/Software_testing).
+    attributes:
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The human readable name of a [test suite](https://en.wikipedia.org/wiki/Test_suite).
+        examples: ['TestSuite1']
+      - id: run.id
+        type: string
+        stability: experimental
+        brief: >
+          The unique identifier of a test suite run.
+        examples: ['123408']
+      - id: run.status
+        type:
+          members:
+            - id: success
+              value: success
+              brief: success
+              stability: experimental
+            - id: failure
+              value: failure
+              brief: failure
+              stability: experimental
+            - id: skipped
+              value: skipped
+              brief: skipped
+              stability: experimental
+            - id: aborted
+              value: aborted
+              brief: aborted
+              stability: experimental
+            - id: timed_out
+              value: timed_out
+              brief: timed_out
+              stability: experimental
+            - id: in_progress
+              value: in_progress
+              brief: in_progress
+              stability: experimental
+        stability: experimental
+        brief: >
+          The status of the test suite run.
+        examples: ['success', 'failure', 'skipped', 'aborted', 'timed_out', 'in_progress']
+      - id: case.name
```

Comment:
That's a good question. I'd be fine with it not living in the suite namespace. The original thinking was that it would live within because cases would be part of a suite, but it's possible that couldn't be 100% guaranteed & would be related to the suite attributes if it was part of it anyway. I'll make this adjustment.

Reply:
i've made this adjustment @lmolkova. resolving this